CVE-2014-0160/data/alice.py

```diff
@@ -0,0 +1,5 @@
+import requests
+c = {
+    'session': 'ultr@_s3cr3t_c00kie'
+}
+requests.get('https://192.168.56.3', cookies=c, verify=False)
```

Comment:
Please do not hardcode IP addresses. You may pass the IP address of ubuntu1 this way:
1. In cvex.yml: "python /opt/alice.py %ubuntu1%"
2. In alice.py: "import sys; requests.get(f"https://{sys.argv[1]}", cookies=c, verify=False)"